custom-connectors/CopilotForFinanceCommunications/apiProperties.json

```diff
@@ -2,13 +2,18 @@
   "properties": {
     "connectionParameters": {
       "token": {
-        "type": "oauthSetting",
+        "type": "oAuthSetting",
```

Comment:
Error; message=Value "oAuthSetting" is not defined in enum.; valuePath=line: 5 position: 30 path: '[0].properties.connectionParameters.token.type'; value=oAuthSetting

Reply:
Thank you very much for resolving review comments, kindly resolve this comment as well. @elkuliko